Formula/lanraragi.rb

```diff
@@ -0,0 +1,161 @@
+require "language/node"
+
+class Lanraragi < Formula
+  desc "Web application for archival and reading of manga/doujinshi"
+  homepage "https://github.com/Difegue/LANraragi"
+  url "https://github.com/Difegue/LANraragi/archive/v.0.6.9-EX.tar.gz"
+  sha256 "157a3ebdb5f132de179b69013aad351333990577bebd31e4b7b80e939b25921d"
+  head "https://github.com/Difegue/LANraragi.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "cpanminus"
+  depends_on "giflib"
+  depends_on "imagemagick@6"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "node"
+  depends_on "openssl@1.1"
+  depends_on "perl"
+  depends_on "redis"
+  uses_from_macos "libarchive"
+
+  resource "Image::Magick" do
+    url "https://cpan.metacpan.org/authors/id/J/JC/JCRISTY/PerlMagick-6.89-1.tar.gz"
+    sha256 "c8f81869a4f007be63e67fddf724b23256f6209f16aa95e14d0eaef283772a59"
+  end
+
+  # libarchive headers from macOS 10.15 source
+  resource "libarchive-headers-10.15" do
+    url "https://opensource.apple.com/tarballs/libarchive/libarchive-72.11.2.tar.gz"
+    sha256 "655b9270db794ba0b27052fd37b1750514b06769213656ab81e30727322e401f"
+  end
+
+  resource "Archive::Peek::Libarchive" do
+    url "https://cpan.metacpan.org/authors/id/R/RE/REHSACK/Archive-Peek-Libarchive-0.38.tar.gz"
+    sha256 "332159603c5cd560da27fd80759da84dad7d8c5b3d96fbf7586de2b264f11b70"
+  end
+
+  # Patch to remove unnecessary exports in installer and fix a bug w/ homebrew overrides
+  patch :DATA
```

Comment:
https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff the build should work without patching before being accepted

Reply:
aight, I'm not planning on making a release right now so this build will wait a bit longer. 👍 

I'll be back ! 😤